src/tagsouppullparser.cpp

```diff
@@ -606,9 +606,6 @@ void TagSoupPullParser::handle_tag()
 
 void TagSoupPullParser::handle_text()
 {
-	if (current_event != Event::START_DOCUMENT) {
```

Comment:
I'd amend the previous commit, 1a9efe0de421d157eaf3f0755eb64126cdff37f5, instead of making a new one.